Lesson6/gokhan/styles.css

```diff
@@ -0,0 +1,158 @@
+styles.css
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+ul {
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 4px 28px;
+  height: 60px;
+  box-shadow: 0 0.25rem 0.75rem rgba(0, 0, 0, .05);
+}
+
+.nav-list {
+  display: flex;
+  gap: 10px;
+  font-size: 12px;
+}
+.btn1 {
+  min-width: 48px;
+  padding: 6px 12px;
+  border-radius: 2px;
+}
+
+.btn2 {
+  border: 1px solid #86B6F6;
+  min-width: 48px;
+  padding: 6px 12px;
+  border-radius: 4px;
+  background-color: #007BF;
+  color: #007BFF;
+}
+
+.btn3 {
+  min-width: 48px;
+  padding: 6px 12px;
+  border-radius: 4px;
+  color: #fff;
+  background-color: #007BFF;
+}
+
+.btn4 {
+  min-width: 48px;
+  padding: 6px 12px;
+  border-radius: 4px;
+  color: #fff;
+  background-color: #007BFF;
+}
+
+.btn-outlined {
+  border: 1px solid #007BFF;
+  border-radius: 2px;
+
+}
+
+.btn-outlined:hover {
+  color: #FFF;
+  background-color: #007BFF;
+  text-decoration: none;
+}
+
+.hero-content {
+  text-align: center;
+  max-width:600px;
+  margin:48px auto 24px;
+}
+
+.cards-container{
+    display: flex;
+    gap: 24px;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.card {
+  min-width: 235px;
+  border: 1px solid rgba(0, 0, 0, 0.125);
+  text-align: center;
+  border-radius: 4px;
+}
+
+.card > h2 {
+ background-color: #DEE2E6;
+ padding: 12px;
+ margin: 0;
+
+}
+.card-content{
+    padding:20px;
+    display:flex;
+    flex-direction: column;
+    gap: 24px;
+    border-top:1px solid rgba(0, 0, 0, 0.125);
+}
+.price{
+    font-size:40px;
+    color:#000;
+}
+.interval{
+    font-size:32px;
+    color:#6c757d;
+}
+
+.card-content > ul {
+  color: #212529;
+}
+
+hr { 
+  margin-top: 4em;
+  margin-left: 140px;
+  margin-right: 140px;
+  border-width: 0.5px;
+  color: #6c757d;
+} 
+
+.footers-container {
+  display:flex;
+  flex-direction: row;
+  justify-content: center;
+  margin: -3em;
+}
+
+.footer > h4{
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+    font-weight: 500;
+}
+.footer{
+  padding: 3rem auto;
+  margin: 50px;
+}
+
+.footer-content{
+  color: #6c757d;
+  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
```

Comment:
I think something is wrong with this font declaration. "Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol"; are not so generic fonts but the they got in the very end of the font declaration so basically there's no chance for them to be displayed. 
https://www.w3schools.com/css/css_font.asp